docs/examples/crawl_sitemap.mdx

```diff
@@ -12,7 +12,7 @@ import CheerioSource from '!!raw-loader!roa-loader!./crawl_sitemap_cheerio.ts';
 import PuppeteerSource from '!!raw-loader!roa-loader!./crawl_sitemap_puppeteer.ts';
 import PlaywrightSource from '!!raw-loader!roa-loader!./crawl_sitemap_playwright.ts';
 
-This example builds a sitemap crawler which downloads and crawls the URLs from a sitemap, by using the <ApiLink to="utils/class/Sitemap">`Sitemap`</ApiLink> utility class provided by the <ApiLink to="utils">`@crawlee/utils`</ApiLink> module.
+Sitemap crawler is a tool that crawl pages of a website, much like search engine crawlers do, in order to gain valuable SEO information. This example builds a sitemap crawler which downloads and crawls the URLs from a sitemap, by using the <ApiLink to="utils/class/Sitemap">`Sitemap`</ApiLink> utility class provided by the <ApiLink to="utils">`@crawlee/utils`</ApiLink> module.
```

Comment:
No sitemap crawler exists (we're building something that could be called that, if you squint hard enough, in this example), I'm afraid this wording is misleading.